src/Chapter3.hs

```diff
@@ -910,6 +1038,22 @@ Implement instances of "Append" for the following types:
 class Append a where
     append :: a -> a -> a
 
+newtype Gold = Gold { goldCount :: Int } deriving (Show)
+
+instance Append Gold where
+  append :: Gold -> Gold -> Gold
+  append g1 g2 = Gold { goldCount = sum $ map goldCount [g1,g2] }
```

Comment:
It's a nice implementation and a good way of using standard functions :+1: Though, allocating an intermediate list can be a bit inefficient. Usually, it's a good idea to implement something simple when possible :slightly_smiling_face: 
```suggestion
  append (Gold g1) (Gold g2) = Gold (g1 + g2)
```

Or are you wanted to use `$` here just to increase the amount of money? :smile: